API/Extensions/UsersExtensions.cs

```diff
@@ -81,7 +84,26 @@ public static string GetStudentId(this ClaimsPrincipal claimsPrincipal, HttpCont
             return studentId;
         }
 
+        /// <summary>
```

Comment:
Fix XML Comment
ClaimsPrincipal is not param

Reply:
as far as the documentation goes i could not find a reference anywhere about extension methods so it is an opinion. while it is not a normal parameter, it is technically a parameter.